modules/network-api/src/main/java/org/apache/ignite/network/ClusterServiceFactory.java

```diff
@@ -28,12 +27,10 @@
      *
      * @param context Cluster context.
      * @param nodeConfiguration Node configuration.
-     * @param nodeFinderSupplier Supplier that provides node finder for discovering the initial cluster members.
      * @return New cluster service.
      */
     ClusterService createClusterService(
         ClusterLocalConfiguration context,
-        ConfigurationManager nodeConfiguration,
-        Supplier<NodeFinder> nodeFinderSupplier
+        ConfigurationManager nodeConfiguration
```

Comment:
I would suggest to replace this parameter with `NetworkView`. That way it would be much easier to mock and it is more descriptive.

Reply:
At this point, the configuration manager has not been started yet, so NetworkView is not available